diff --git a/node_modules/@redux-offline/redux-offline/typings.d.ts b/node_modules/@redux-offline/redux-offline/typings.d.ts
index ffa5006..d91c70f 100644
--- a/node_modules/@redux-offline/redux-offline/typings.d.ts
+++ b/node_modules/@redux-offline/redux-offline/typings.d.ts
@@ -81,7 +81,7 @@ declare module '@redux-offline/redux-offline/lib/types' {
     offlineStateLens: (
       state: any,
     ) => { get: OfflineState, set: (offlineState?: OfflineState) => any };
-    persist: (store: any) => any;
+    persist: boolean;
     persistAutoRehydrate: (config?: { [key: string]: any }) => (next: any) => any;
     persistCallback: (callback?: any) => any;
     persistOptions: { [key: string]: any };
