diff --git a/node_modules/react-native-image-filter-kit/android/build.gradle b/node_modules/react-native-image-filter-kit/android/build.gradle
index 6546ccf..8714cc9 100755
--- a/node_modules/react-native-image-filter-kit/android/build.gradle
+++ b/node_modules/react-native-image-filter-kit/android/build.gradle
@@ -2,24 +2,28 @@ apply plugin: 'com.android.library'
 
 def _ext = rootProject.ext;
 
-def _reactNativeVersion = _ext.has('reactNative') ? _ext.reactNative : '+';
-def _compileSdkVersion = _ext.has('compileSdkVersion') ? _ext.compileSdkVersion : 29;
-def _buildToolsVersion = _ext.has('buildToolsVersion') ? _ext.buildToolsVersion : '29.0.3';
-def _minSdkVersion = _ext.has('minSdkVersion') ? _ext.minSdkVersion : 21;
-def _targetSdkVersion = _ext.has('targetSdkVersion') ? _ext.targetSdkVersion : 29;
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+// def _reactNativeVersion = _ext.has('reactNative') ? _ext.reactNative : '+';
+// def _compileSdkVersion = _ext.has('compileSdkVersion') ? _ext.compileSdkVersion : 29;
+// def _buildToolsVersion = _ext.has('buildToolsVersion') ? _ext.buildToolsVersion : '29.0.3';
+// def _minSdkVersion = _ext.has('minSdkVersion') ? _ext.minSdkVersion : 21;
+// def _targetSdkVersion = _ext.has('targetSdkVersion') ? _ext.targetSdkVersion : 29;
 
 android {
-    compileSdkVersion _compileSdkVersion
-    buildToolsVersion _buildToolsVersion
+    compileSdkVersion safeExtGet('compileSdkVersion', 31)
+    buildToolsVersion safeExtGet('buildToolsVersion', '31.0.0')
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
     }
 
     defaultConfig {
-        minSdkVersion _minSdkVersion
-        targetSdkVersion _targetSdkVersion
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 31)
         versionCode 1
         versionName "1.0"
     }
@@ -29,7 +33,10 @@ android {
 }
 
 dependencies {
-    compileOnly "com.facebook.react:react-native:${_reactNativeVersion}"
+    compileOnly "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    implementation "com.android.support:appcompat-v7:${safeExtGet('supportLibVersion', '+')}"
+    
+    // compileOnly "com.facebook.react:react-native:${_reactNativeVersion}"
     //noinspection GradleDynamicVersion
     compileOnly "com.parse.bolts:bolts-tasks:+"
     implementation 'com.github.iyegoroff:ReflectUtils:1.2.1'
