diff --git a/node_modules/redux-persist/types/persistStore.d.ts b/node_modules/redux-persist/types/persistStore.d.ts
index bdcf2f3..3bd0ea5 100644
--- a/node_modules/redux-persist/types/persistStore.d.ts
+++ b/node_modules/redux-persist/types/persistStore.d.ts
@@ -1,5 +1,5 @@
 declare module "redux-persist/es/persistStore" {
-  import { Store } from 'redux';
+  import { Store, Action, AnyAction } from 'redux';
   import { PersistorOptions, Persistor } from "redux-persist/es/types";
 
   /**
@@ -9,7 +9,7 @@ declare module "redux-persist/es/persistStore" {
    * @param callback bootstrap callback of sort.
    */
   // tslint:disable-next-line: strict-export-declare-modifiers
-  export default function persistStore(store: Store, persistorOptions?: PersistorOptions | null, callback?: () => any): Persistor;
+  export default function persistStore<S = any, A extends Action<any> = AnyAction>(store: Store<S, A>, persistorOptions?: PersistorOptions | null, callback?: () => any): Persistor;
 }
 
 declare module "redux-persist/lib/persistStore" {
